sdk/core/azure-core-metrics-opentelemetry/src/main/java/com/azure/core/metrics/opentelemetry/OpenTelemetryAttributes.java

```diff
@@ -22,10 +22,10 @@ private static Map<String, String> getMappings() {
         Map<String, String> mappings = new HashMap<>();
         // messaging mapping, attributes are defined in com.azure.core.amqp.implementation.ClientConstants and in EventHubs, ServiceBus
         // metric helpers
-        mappings.put("status", "otel.status_code");
+        mappings.put("status", "error.type");
```

Comment:
Would we ever enable the ability for users to specify which convention so use? This would allow the SDKs to match their environment rather than following whatever the latest specification the SDKs are using.

Reply:
That was the plan, but I don't think we will now.
The semconv version is for us to specify which version we're using. (e.g. `error.type` is defined in version 1.23.0).

If a new attribute `error.foo` is introduced in 1.42.0 we won't start reporting it until our code is updated. And then we'd just report version 1.42.

It's totally fine for stable attributes, and is only problematic if otel has any breaking changes.
We'll keep unstable attributes behind the feature flag (not done yet, but the whole package is still in beta) and will keep breaking them until they stabilize.